### PR TITLE
Feature/python/new_demos

### PR DESCRIPTION
**Pull request by michaelhly**
## Description
New demos to demonstrate usage of python packages.

## Testing instructions
./setup.py/launch_kit
./setup.py/test

## Types of changes

New feature (non-breaking change which adds functionality) 

## Checklist:

-   [X] Prefix PR title with `[WIP]` if necessary.
-   [X] Add tests to cover changes as needed.
-   [X] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

**Comment by michaelhly** [comment]
Blocked until https://github.com/0xProject/0x-monorepo/pull/1730 is merged.

File: python-packages/demos/src/demo_sra_client.py [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                                     'maker_asset_data': '0xf47261b0000000000000000000000000d0a1e359811322d97991e03f863a0c30c2cf029c',
+                                     'maker_fee': '0',
+                                     'salt': '2362734632784682376287462',
+                                     'sender_address': '0x0000000000000000000000000000000000000000',
+                                     'taker_address': '0x0000000000000000000000000000000000000000',
+                                     'taker_asset_amount': '500000000000000000000',
+                                     'taker_asset_data': '0xf47261b00000000000000000000000002002d3812f58e35f0ea1ffbf80a75a38c32175fa',
+                                     'taker_fee': '0'}}]},
+     'bids': {'records': []}}
+    """  # noqa: E501
```
Comment (feuGeneA) [inline python-packages/demos/src/demo_sra_client.py]:
```suggestion
    """  # noqa: E501 (line too long)
```

File: python-packages/demos/src/index.rst [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+Setup
+-----
+Install the sra-client package with pip:
+
+`pip install https://github.com/0xproject/0x-monorepo/python-packages/sra_client`:code:
+
+To interact with a 0x-relayer, you need the HTTP endpoint of the relayer you'd like to 
+connect to (i.e. https://api.radarrelay.com/0x/v2). 
+
+For local testing one use the `0x-launch-kit
```
Comment (feuGeneA) [inline python-packages/demos/src/index.rst]:
```suggestion
For local testing one can use the `0x-launch-kit 
```

File: python-packages/demos/src/index.rst [hunk 3]
```diff
@@ -0,0 +1,68 @@
+.. source for the sphinx-generated build/docs/web/index.html
```
Comment (feuGeneA) [inline python-packages/demos/src/index.rst]: I think this content belongs in the `sra_client` module docstring, and the sphinx generation should happen there too.

**Pull request closed without merging**